client/index.html

```diff
@@ -59,7 +59,7 @@
     <div id="root"></div>
     <script
       async
-      src="static/obsoleteBrowsers.js"
+      src="/static/obsoleteBrowsers.js"
```

Comment:
I believe the `/` prefix was only for local testing, so might need to reverse this back. Thank you!